tests/test_fakes.py

```diff
@@ -875,13 +875,15 @@ def test_show_objects(dcur: snowflake.connector.cursor.SnowflakeCursor):
             "kind": "TABLE",
             "database_name": "DB1",
             "schema_name": "SCHEMA1",
+            "comment": None,
```

Comment:
snowflake doesn't show the `comment` field for a **terse** summary. i'd like to make sure terse doesn't have extra columns that someone might unexpectedly come to depend on.

`comment` appears as part of `show objects` which has this complete set of fields:

created_on, name, database_name, schema_name, kind, comment, cluster_by, rows, bytes, owner, retention_time, owner_role_type, budget